gradle/libs.versions.toml

```diff
@@ -6,10 +6,10 @@ okio = "3.2.0"
 serialization = "1.4.1"
 kotlinx-datetime = "0.4.0"
 kotlinx-coroutines = "1.6.4"
-kotlin-wrappers = "1.0.0-pre.433"
+kotlin-wrappers = "1.0.0-pre.452"
 # There is a regression https://github.com/spring-projects/spring-data-jpa/issues/2683 in spring-data-jpa 2.7.5.
 # Staying on 2.7.4 until there is a fix available.
```

Comment:
I guess it can be removed (I'm about the comment)

Reply:
and what do you think about renovate.json -- do we need to clean exclusion for 2.7.4 there?